src/main/java/friday/main/Friday.java

```diff
@@ -0,0 +1,137 @@
+package friday.main;
+
+import java.io.IOException;
+
+import friday.parser.Parser;
+import friday.storage.Storage;
+import friday.task.TaskList;
+import friday.task.Todo;
+import friday.task.Deadline;
+import friday.task.Event;
+import friday.ui.Ui;
+
+/**
+ * The main class for the Friday task management application.
+ * Friday manages tasks such as Todos, Deadlines, and Events, providing a simple command-line interface for users.
+ * Tasks are stored in a data file, and users can interact with Friday by entering commands.
+ */
+public class Friday {
+    private static final String DATA_FILE_PATH = "./src/main/java/data/Friday.txt";
```

Comment:
I like that your constant names are in uppercase with underscores 👍

Reply:
Thx!